### PR TITLE
Tentative: Reorganise e2e run e2e

### PR DESCRIPTION
**Pull request by ytvnr**
## Description

Try to re-organise the E2E testing `api-tests`.

Proposition is to have a tree representing the feature we are testing, and inside a feature, precise if it's done through `mapi-v1` or `mapi-v2`.

It also gets rid of folder `use-case-test` which had a purpose back in time (regression test pack) but is now bringing confusion about where to write a new test.

image

---

📚&nbsp;&nbsp;View the storybook of this branch [here](https://612657caa8e859003a8a6430-qvikfgnwqw.chromatic.com)

### CODE REVIEW

File: gravitee-apim-e2e/api-test/src/apis/debug-mode/mapi-v1/call-my-api-including-query-params-headers-body-and-view-debug-session-with-proper-info.spec.ts [hunk 1]
```diff
@@ -14,21 +14,21 @@
  * limitations under the License.
  */
 import { afterAll, beforeAll, describe, expect } from '@jest/globals';
-import { OrganizationEntityToJSON } from '@gravitee/management-webclient-sdk/src/lib/models/OrganizationEntity';
+import { OrganizationEntityToJSON } from '../../../../../lib/management-webclient-sdk/src/lib/models/OrganizationEntity';
```
Comment (ctschacher) [inline gravitee-apim-e2e/api-test/src/apis/debug-mode/mapi-v1/call-my-api-including-query-params-headers-body-and-view-debug-session-with-proper-info.spec.ts]: What's the reason for not using the TS aliases, by the way?

**Review by ctschacher — APPROVED** [review]
Looks good to me

File: gravitee-apim-e2e/api-test/src/apis/debug-mode/mapi-v1/call-my-api-including-query-params-headers-body-and-view-debug-session-with-proper-info.spec.ts (hunk 1 above)
Comment (ytvnr) [reply gravitee-apim-e2e/api-test/src/apis/debug-mode/mapi-v1/call-my-api-including-query-params-headers-body-and-view-debug-session-with-proper-info.spec.ts]: no reason, intellij did it automatically
i'll fix that in another commit ;)

**Review by Okhelifi — APPROVED** [review]
👏🏼

**Pull request merged**